dev/TabView/TabView.cpp

```diff
@@ -590,14 +606,21 @@ void TabView::OnItemsChanged(winrt::IInspectable const& item)
         m_tabItemsChangedEventSource(*this, args);
 
         int numItems = static_cast<int>(TabItems().Size());
+        const auto listViewInnerSelectedIndex = m_listView.get().SelectedIndex();
+        auto selectedIndex = SelectedIndex();
+        
+        if (selectedIndex != listViewInnerSelectedIndex && listViewInnerSelectedIndex != -1)
+        {
+            SelectedIndex(listViewInnerSelectedIndex);
+            selectedIndex = SelectedIndex();
```

Comment:
>SelectedIndex [](http://example.com/codeflow?start=28&length=13)

nit: selectedIndex = listViewInnerSelectedIndex;

avoid using the DP getter if you don't need to.

Reply:
will do